apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -65,7 +65,12 @@ function SectionProgressSelector({
 
   const toggleV1OrV2Link = () => (
     <div className={styles.toggleViews}>
-      <Link type="primary" size="s" onClick={onShowProgressTableV2Change}>
+      <Link
+        type="primary"
+        size="s"
+        onClick={onShowProgressTableV2Change}
+        id="ui-test-link-to-new-progress-view"
```

Comment:
```suggestion
        id="ui-toggle-progress-views"
```
This button will also turn from new to old views.

Reply:
Good point - changed!